### DISCUSSION TITLE
Unable to create an ec2 instance

### DISCUSSION THREAD

**Discussion by kushalmusale** [learndevopsonline/learndevopsonline#15] [Q&A] [answered]
Hi Raghu, 

Since yesterday I am not able to create an instance using Centos-8-DevOps-Practice AMI. 
neither getting any message for not creating an instance

Any idea on why am I facing this issue

**Comment by chandrashekarsubuddi** [comment, marked as answer]
i think you need to unselect Elastic Graphics from the add additional tags under resources then it will work

**Reply by kushalmusale** [reply]
Thanks @chandrashekarsubuddi, I am able create an instance now by not selecting Elastic Graphics. Thank you

**Reply by chandrashekarsubuddi** [reply]
@kushalmusale Thank you
